src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -122,18 +126,36 @@ GHRelease wrap(GHRepository owner) {
      * http://stackoverflow.com/questions/12361090/server-name-indication-sni-on-java but involve more complicated
      * handling of the HTTP requests to github's API.
      *
+     * @param file file to upload
+     * @param contentType content type
+     * @param uploadUrlPrefix URL prefix for uploading; for example, for github.com it's https://uploads.github.com
+     * @return instance of GHAsset uploaded
+     * @since 1.75
      * @throws IOException
      */
-    public GHAsset uploadAsset(File file, String contentType) throws IOException {
+    public GHAsset uploadAsset(File file, String contentType, String uploadUrlPrefix) throws IOException {
         Requester builder = new Requester(owner.root);
 
-        String url = format("https://uploads.github.com%s/releases/%d/assets?name=%s",
-                owner.getApiTailUrl(""), getId(), file.getName());
+        String url = format("%s%s/releases/%d/assets?name=%s",
+                            uploadUrlPrefix, owner.getApiTailUrl(""), getId(), file.getName());
         return builder.contentType(contentType)
                 .with(new FileInputStream(file))
                 .to(url, GHAsset.class).wrap(this);
     }
 
+    /**
+     * upload the release to defaul upload URL on github.com
+     *
+     * @param file file to upload
+     * @param contentType content type
+     * @return instance of GHAsset uploaded
+     * @throws IOException
+     */
+    public GHAsset uploadAsset(File file, String contentType) throws IOException {
```

Comment:
yep!
You can try `defaultIfEmpty()` from commons-lang (if available) instead of `uploadUrlPrefix != null ? uploadUrlPrefix : DEFAULT_UPLOAD_URL;`

and call `return this.uploadAsset(file, contentType, DEFAULT_UPLOAD_URL);`

also please reflect in javadoc, that third param may be null and default value will be used instead